packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -720,6 +742,18 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
     )
   }
 
+  private renderLabel(getLabelProps: (options?: GetLabelPropsOptions) => any) {
```

Comment:
can we move this in a separate file?

this one is getting very big; let's try be proactive about the upcoming refactoring (splitting logic to separate files)